R/plot_jags_densitydx.R

```diff
@@ -76,18 +60,23 @@ plot_jags_dens <- function(data,
         # Changing parameter name to reflect the input
         dplyr::mutate(Parameter = paste0("iso = ", j, ", parameter = ",
                                          .data$Parameter_sub, ", strat = ",
-                                         i),
-                      value = log(.data$value))
+                                         i))
       # Assigning attributes, which are needed to run ggs_density
       attributes(visualize_jags_plot) <- c(attributes(visualize_jags_plot),
                                            attributes_jags)
       # Creating density plot
       densplot <- ggmcmc::ggs_density(visualize_jags_plot) +
         ggplot2::theme_bw() +
-        ggplot2::labs(x = "log(value)")
+        ggplot2::labs(x = "log(value)") +
```

Comment:
```suggestion
        ggplot2::labs(x = "parameter value") +
```
(or leave it as whatever `ggs_density()` defaults to?)